apps/condo/domains/notification/helpers/compareMobileAppVersion.js

```diff
@@ -0,0 +1,28 @@
+function compareMobileAppVersions (ver1, ver2) {
```

Comment:
In compare functions it's common practice to name variables `(a, b)` (which is not readable imho) or `(lhs, rhs)`, which I think is good. (left hand side and right hand side)